voluncheer/opportunityboard/management/commands/load_categories.py

```diff
@@ -0,0 +1,47 @@
+"""The load_categories command injects missing categories into the database."""
+
+import json
+
+from django.core.management.base import BaseCommand
+
+from opportunityboard.models import Category
+from opportunityboard.models import Subcategory
+from opportunityboard.models import Subsubcategory
+
+# JSON model type representations.
+_CATEGORY = "opportunityboard.category"
+_SUB_CATEGORY = "opportunityboard.subcategory"
+_SUB_SUB_CATEGORY = "opportunityboard.subsubcategory"
+
+
+class Command(BaseCommand):
+    help = "Loads missing categories for the opportunity board."
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--category_data",
+            default="opportunityboard/fixtures/category_data.json",
+            help="The email address of the super user to create.",
```

Comment:
I think this made it over from the su command.

Reply:
Hah, failed copy/pasta. Thanks for noticing and mentioning. Fixed.